lib/meetup/latest-meetup.js

```diff
@@ -0,0 +1,28 @@
+// jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+import moment from 'moment';
+import Meetup from 'meetup-api';
+
+const meetup = Meetup({ key: process.env.MEETUP_API_TOKEN });
+
+export default function getLatestMeetup(cb) {
+  const meetupReqParams = {
+    group_urlname: process.env.MEETUP_GROUP_URLNAME,
+    page: 1,
+    text_format: 'plain'
+  };
+  meetup.getEvents(meetupReqParams, (err, res) => {
```

Comment:
This is fine, though I generally prefer the following style as I find it easier to read and add to.

```js
  if (err) return cb(err);

  if (!res.results) {
    return cb(null, 'No meetups found!');
  };

  let nextEvent = res.results[0];
  let nextEventDate = moment(nextEvent.time).format('h:mmA dddd Do MMM');
  cb(null, `Next meetup: ${nextEvent.name} - ${nextEventDate} @ ${nextEvent.venue.name} (${nextEvent.venue.address_1})`);
```